scilpy/tracking/local_tracking.py

```diff
@@ -17,8 +17,7 @@
 
 
 def track(tracker, mask, seed, param, compress=False,
-          compression_th=0.1, nbr_processes=1,
-          pft_tracker=None, save_seeds=False):
+          compression_th=0.1, nbr_processes=1, save_seeds=False):
```

Comment:
Could we remove `compress` and only use `compression_th`, setting it to `None` to disable compression?